### PR TITLE
Default new group's captain to creator

### PR DESCRIPTION
**Pull request by bion**
Since non-admins can create groups, we associate newly created groups with their creator as the group's captain.

Also made the profile endpoint return the user account ID and whether or not they're an admin. Over time we'll probably put other user-specific useful things for the UI in the profile.

### CODE REVIEW

File: db/migrations/20210308070729-add-captain-to-group.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkDelete('Shipments', {})
+    await queryInterface.bulkDelete('Groups', {})
```
Comment (deammer) [inline db/migrations/20210308070729-add-captain-to-group.js]: Question: why is this necessary?

File: src/apolloServer.ts [hunk 2]
```diff
@@ -7,17 +7,25 @@ import depthLimit from 'graphql-depth-limit'
...
+} from './authenticateRequest'
 
 export type Context = {
   auth: Auth
 }
 
+export type AuthenticatedContext = {
+  auth: AuthenticatedAuth
+}
+
```
Comment (deammer) [inline src/apolloServer.ts:23]: 💯

**Review by deammer — APPROVED** [review]
Looks good to me!

File: db/migrations/20210308070729-add-captain-to-group.js (hunk 1 above)
Comment (bion) [reply db/migrations/20210308070729-add-captain-to-group.js]: Eh, it probably isn't. Everyone is going to need to reset their DBs anyway. I'll remove it.
